azure-pipelines.yml

```diff
@@ -1,19 +1,127 @@
-# Starter pipeline
-# Start with a minimal pipeline that you can customize to build and deploy your code.
-# Add steps that build, run tests, deploy, and more:
-# https://aka.ms/yaml
+variables:
+  CurrentSemanticVersionBase: '7.0.0'
```

Comment:
I think we wanted to match the current Windows Community Toolkit versioning and thus start at 7?

Reply:
Up to you folks, I know we're probably going to be on different shipping cycles, so not sure how feasible it's going to be to always ship in-sync?

We're currently on 7.0 released and moving towards our 7.1 release for this month. Probably plan a 7.2 in time for .NET 6/WinUI 2.7/WinUI 3 1.0 though no idea if all those milestones align 🤷‍♂️? Lots of moving parts and dependencies now! 😋

I know after the 7.1 for WCT we'll start splitting out the .NET Community Toolkit bits to their own repo in September-ish, so that'd be part of that 7.2 drop as well is that being independent.

Love to know your thoughts. If you think you'll be snapping to WinUI 3 releases, then that does make it a bit easier assuming WinUI 2.x and WinUI 3 releases are relatively synced in terms of scheduling in the future (as we'll be updating the WCT more often to match the WinUI 2.x releases moving forward now that we take them as a dependency).

Reply:
Definitely true. We might want to discuss this a bit further. @brminnick any thoughts on this as well? Maybe we should follow up on Teams

Reply:
I think it's best to keep our versioning separate. 

Since the MAUI Toolkit doesn't have a dependency on the Windows Toolkit, there's not much need for them to keep their versions in lock-step. Keeping the version numbers the same would just add more overhead for both toolkits because each time we release a new version of the Maui Toolkit, the Windows Toolkit would need to release a new version and vice-versa.

We kept the Xamarin.CommunityToolkit NuGet Version in lock-step with the Xamarin.CommunityToolkit.Markdown NuGet and we ended up releasing "new versions" of the Markup NuGet that were exactly the same as the previous version (no code in the Markup library had changed).

I vote that we start both `CommunityToolkit.Maui` `CommunityToolkit.Maui.Markup` at v1.0.0-pre1 and let them increase their version numbers independently from each other and independently from Windows Toolkit's versioning.

Reply:
Makes sense to me! Reset it 1 for now, we can always decide otherwise before we merge 😬